x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_events_query.ts

```diff
@@ -27,6 +28,7 @@ interface BuildEventsSearchQuery {
   secondaryTimestamp: TimestampOverride | undefined;
   trackTotalHits?: boolean;
   additionalFilters?: estypes.QueryDslQueryContainer[];
+  overrideBody?: OverrideBodyQuery;
```

Comment:
@nkhristinin could we add some js docs comments explaining what this property is used for? Something like this suggestion?

```suggestion
  /**
    * overrideBody allows the search after to ignore the _source property of the result, thus reducing the size of the response and increasing the performance of the query.
    */
  overrideBody?: OverrideBodyQuery;
```

Reply:
Hey, thanks! Added comment to a single search function.